### ISSUE TITLE
KmzTrackImport Context.

### ISSUE THREAD

**Issue by rgmf** [OpenTracksApp/OpenTracks#681]
KmzTrackImport has a Context like a member but some of its method receive the context like argument. Does it have any sense? Why not to use the Context on these methods?

The constructor:

```java
public KmzTrackImporter(Context context, Uri uriFile) {
        this.context = context;
        this.uriKmzFile = uriFile;
    }
```

One of its method (for example):

```java
private void cleanImport(Context context, Track.Id trackId) {
```

**Comment by dennisguse** [comment]
Best guess: some leftovers that were forgotten and it is actually confusing as the parameter actually suggests that there could be another context.

**Comment by dennisguse** [comment]
Fixed in e09b04308c5d62173458e7f803d39a2ba33bab91
